android/app/src/main/res/values/styles.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+    <!-- Forcing the light theme for the entire app -->
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
Do you know the difference between that and        ` <item name="android:forceDarkAllowed">false</item>` ?https://github.com/facebook/react-native/issues/27876

Reply:
@Crash-- from what I understand we should use both to be sure. They seem to do the same thing but in a different way (to me). It's possible some devices may work or not in one mode or another.

Reply:
updated